.github/workflows/ci.yml

```diff
@@ -38,21 +38,6 @@ jobs:
       - name: Scan for security vulnerabilities in JavaScript dependencies
         run: bin/importmap audit
 
-  lint:
```

Comment:
considering this is run when deploys happen, i removed this and created a new file instead.

Reply:
this is still called the test and linter but it can be renamed to something like: security scans and tests, but i didn’t want to be too presumptuous.